### ISSUE TITLE
Don't verify checksum of GothicMod.exe for G2

### ISSUE THREAD

**Issue by dosinabox** [kirides/GD3D11#99]
Was previously done in SaiyansKing's fork but unfortunately lost or ignored when merging to this project:
https://github.com/SaiyansKing/GD3D11/issues/18

Please do the same change here too.

**Comment by ShamanInCap** [comment]
I agree. No one will play it today without fixes, that has become necessary
